test/server/lib/HostedStorageManager.ts

```diff
@@ -1000,15 +1004,23 @@ describe('HostedStorageManager', function() {
       });
       await docWorkerMap.setWorkerAvailability(workerId, true);
 
+      class Callbacks implements HostedStorageCallbacks {
+          private _internalSet(metadata: any) {}
+          public async setDocsMetadata(metadata: any) {
+            // Access `this` to ensure the callback is being bound correctly when passed to HostedMetadataManager.
+            this._internalSet(metadata);
+          }
+          public async getDocFeatures(docId: any) {
+            return undefined;
+          }
+      }
+
       defaultParams = [
         tmpDir,
         workerId,
         false,
         docWorkerMap,
-        {
-          setDocsMetadata: async (metadata) => {},
-          getDocFeatures: async (docId) => undefined,
-        },
+        new Callbacks(),
```

Comment:
Don't think this is supposed to be in here, I think this came out of a git stash. Will remove.